parameterhandler/README.md

```diff
@@ -1,20 +1,21 @@
 # Parameter handler in ACAPs
-This directory aims to clarify how parameter handling can be done in ACAP applications. The options differ depending on which SDK you use to build your application.
+This directory aims to clarify how parameter handling can be done in ACAP applications.
 
-All Axis cameras comes with a built in parameter handler, in this are a lot of configuration options for the camera. Earlier SDKs also allowed third party applications to add their parameters to the camera's parameter handler. The parameter handler is available e.g. using the `param.cgi` VAPIX API or the `parhandclient` command using SSH. Many features has during the last few years been migrated to separate APIs instead of the parameter handler, this allows for better validation of values and synchronization of changes.
+All Axis cameras comes with a built in parameter handler, in this handler is a lot of configuration options for the camera. In earlier SDKs the standard way for third party applications to handle configuration was to add their parameters to the camera's parameter handler. This option was later removed (from the first release of SDK 4) but it was then reintroduced again. The parameter handler is available using e.g. the `param.cgi` VAPIX API or the `parhandclient` command using SSH. Here you can see and change both the cameras own parameters and your applications parameters. Many features has during the last few years been migrated to separate APIs instead of the parameter handler, this allows for better validation of values and synchronization of changes. The same is true for ACAPs.
```

Comment:
All Axis cameras come*

Reply:
both the camera's* own parameters and your application's* parameters

or

both the cameras'* own parameters and your applications'* parameters

Reply:
Many features have* during the last few years